index.html

```diff
@@ -115,7 +115,7 @@
 }
 
 function fontSizeChanged() {
-    fontSize = fontSizeSlider.value;
+    fontSize = parseInt(fontSizeSlider.value, 10);
```

Comment:
Sorry for that, I checked and as the slider is configured to only output integers parseFloat is not needed...!